MELA/src/TUtil.cc

```diff
@@ -5271,20 +5273,22 @@ double TUtil::MadgraphMatEl(
     );
   
   const int nPDG = mela_event.pDaughters.size() + mela_event.pAssociated.size() + mela_event.pMothers.size();
-  vector<int> pdgs(nPDG);
-  vector<vector<double>> p(nPDG, vector<double>(4));
-
   TLorentzVector MomStore[mxpart]; // Mom (in natural units) to compute alphaS
   for (int i = 0; i < mxpart; i++) MomStore[i].SetXYZT(0, 0, 0, 0);
-  // if(verbosity >= TVar::DEBUG) MELAout << "Boost vector of " << boostVec.Px() << " " << boostVec.Py() << " " << boostVec.Pz() << endl;
+
+  // vector<int> pdgs(nPDG);
+  // vector<vector<double>> p(nPDG, vector<double>(4));
+  
+  int pdgs[nPDG];
+  double* p = new double[4*nPDG];
```

Comment:
Overhaul the creation of the input 4-vectors to skip the first vector-quantity part to increase speed